source/material_model/steinberger.cc

```diff
@@ -769,6 +769,15 @@ namespace aspect
           out.entropy_derivative_temperature[i] = 0;
           for (unsigned int c=0; c<in.composition[i].size(); ++c)
             out.reaction_terms[i][c]            = 0;
+
+          // fill seismic velocities outputs if they exist
+          SeismicAdditionalOutputs<dim> *seismic_out = out.template get_additional_output<SeismicAdditionalOutputs<dim> >();
+
+          if (seismic_out != NULL)
+            {
+              seismic_out->vp[i] = seismic_Vp(in.temperature[i], in.pressure[i], in.composition[i], in.position[i]);
+              seismic_out->vs[i] = seismic_Vs(in.temperature[i], in.pressure[i], in.composition[i], in.position[i]);
```

Comment:
here, for example, we're going to do additional work even if we're currently just assembling the matrix